order-service/order-service.ts

```diff
@@ -0,0 +1,201 @@
+import { Tx, Pair, Position, OrderConfig } from '../types/common.js';
+import {
+  OrderSide,
+  OrderType,
+  BECH32_PREFIX,
+  LocalWallet,
+  SubaccountClient,
+  Network,
+  CompositeClient,
+  IndexerClient,
+} from '@dydxprotocol/v4-client-js';
+
+// order factory function
+const createOrderService = async (mnemonic: string, network: Network) => {
+  const wallet = await LocalWallet.fromMnemonic(mnemonic, BECH32_PREFIX);
+  const subaccount = new SubaccountClient(wallet, 0);
+  const client = await CompositeClient.connect(network);
+
+  const indexer_client = new IndexerClient(network.indexerConfig);
+
+  return {
+    placeLimitOrder: (
+      pair: Pair,
+      side: OrderSide,
+      price: number,
+      size: number,
+      config: OrderConfig,
```

Comment:
This config should be optional (if possible) with reasonable defaults. We can talk about these "reasonable defaults".